### PR TITLE
Update to use latest Exocom/ExoRelay/ExoService

### PR DESCRIPTION
**Pull request by hugoesthere**
End-to-end tests pass

@kevgo

### CODE REVIEW

File: cli/features/tutorial.feature [hunk 1]
```diff
@@ -28,7 +28,7 @@ Feature: Following the tutorial
       | Name of the application to create | todo-app           |
       | Description                       | A todo application |
       | Initial version                   |                    |
-      | ExoCom version                    | 0.16.0             |
+      | ExoCom version                    | latest             |
```
Comment (kevgo) [inline cli/features/tutorial.feature]: Why `latest` here? Can we use a real version?

Comment (hugoesthere) [reply cli/features/tutorial.feature]: We can, but each time we update ExoCom we'd have to change the version number in the feature tests and all of the example application templates.

It seemed hacky to me when I had to go through the whole monorepo to find and replace `0.16.0` with `0.16.1` when I had published a patch of ExoCom. Plus a new version of ExoCom could be released and our tests would be running against an older version. The developer can certainly define which version of ExoCom they want in their application.yml though.

Comment (hugoesthere) [reply cli/features/tutorial.feature]: Nevermind, `latest` could mean the user pulled the `latest` from 3 weeks ago. I'll change this back to a hard version number

**Comment by hugoesthere** [comment]
@kevgo good for another review

File: cli/features/tutorial.feature (hunk 1 above)
Comment (kevgo) [reply cli/features/tutorial.feature]: I don't expect us to update Exocom that much. If it becomes a problem, we can automate the search and replace. It's pretty straightforward in Vim..  :)

**Review by kevgo — APPROVED** [review]

**Comment by kevgo** [comment]
CI is still failing though..

**Comment by hugoesthere** [comment]
Yeah, sometimes the end-to-end test fails on the step executing `exo test` on an example todo app. So technically it's an issue with the tests we wrote for that example app and not the `exo` commands. I've been having trouble reproducing that error on my machine, and sometimes on Travis if I rebuild it goes away.

Regardless, Travis will still be red right now because some of the other sub-projects are failing - which I'm working on fixing.

**Pull request closed without merging**